Update B002 and make it more abstract. (#19)
qualiticcommunity/qualiticcommunity.github.io: catalog/build_ci/B002.md

### B002

|**Code**           | **B002** |

> |**Title**          | Select and Enforce Code Style for Main Programming Languages. |

Reviewer: ```suggestion
|**Title**          | Code Style Checking for Main Programming Languages. |
```
I prefer the item title not be a whole sentence as it was before.

Reviewer 2: I disagree with the suggestion since the previous seems more clear and comprehensive, IMHO.

Reviewer 2: As a compromise, I suggest this if you agree:
```suggestion
|**Title**          | Enforcing a Specific Code Style for Main Programming Languages. |
```

Author: I replaced the title with @emranbm suggestion.
